<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LP0ER4S" failures="2" tests="9" name="Edge_Test" time="71.813" errors="0" timestamp="11 Jun 2025 14:39:42 GMT">
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyForgotPassword" time="2.338"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyFunctionalityForGooglePlayTest" time="2.469">
    <failure type="java.lang.AssertionError" message="expected [https://play.google.com/store/apps/details?id=com.adactin.education.hotelbooking] but found [https://adactinhotelapp.com/index.php]">
      <![CDATA[java.lang.AssertionError: expected [https://play.google.com/store/apps/details?id=com.adactin.education.hotelbooking] but found [https://adactinhotelapp.com/index.php]
at com.adactin.testcases.LoginPageTest.verifyFunctionalityForGooglePlayTest(LoginPageTest.java:100)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- verifyFunctionalityForGooglePlayTest -->
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyFunctionalityForiOSTest" time="5.455">
    <failure type="java.lang.AssertionError" message="expected [https://testflight.apple.com/join/Ai3nVPMD] but found [https://adactinhotelapp.com/index.php]">
      <![CDATA[java.lang.AssertionError: expected [https://testflight.apple.com/join/Ai3nVPMD] but found [https://adactinhotelapp.com/index.php]
at com.adactin.testcases.LoginPageTest.verifyFunctionalityForiOSTest(LoginPageTest.java:76)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- verifyFunctionalityForiOSTest -->
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyLoginFunctionality" time="4.344"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyLoginFunctionality" time="3.259"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyLoginFunctionality" time="3.108"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyLogoTest" time="0.08"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyNewUserRegisterHereLink" time="1.188"/>
  <testcase classname="com.adactin.testcases.LoginPageTest" name="verifyTitleTest" time="0.017"/>
</testsuite> <!-- Edge_Test -->
